schemas/hotelSchema.js

```diff
@@ -0,0 +1,56 @@
+const Joi = require('joi')
+const mongoose = require('mongoose')
+const JoiPhoneNumer = Joi.extend(require('joi-phone-number'))
+import { Room } from './roomSchema'
+import { clientRateSchema } from './rateSchema'
+
+const hotelSchema = new mongoose.Schema({
+  ownerId: {
+    type: mongoose.Types.ObjectId,
+    ref: 'User',
+  },
+  localization: [
```

Comment:
Why this is an array? Will hotel have more localizations?

Reply:
Yeeah and I was wondering why prettier formated that this way 😄